libraries/core_libs/consensus/src/transaction/transaction_queue.cpp

```diff
@@ -48,6 +48,9 @@ SharedTransactions TransactionQueue::getOrderedTransactions(uint64_t count) cons
     it->second.first++;
   }
   while (true) {
```

Comment:
why not put it here ? 
`while(!head_transactions.empty())`

Reply:
Done